aries-site/src/examples/components/card/ActivitiesNavigationalCards.js

```diff
@@ -37,6 +37,7 @@ export const ActivitiesNavigationalCards = ({ heading = true }) => {
               />
             }
             level={3}
+            skeleton={skeleton}
```

Comment:
I know you didn't touch it in this code, but can we sentence case "Activities and tasks" and use "and" instead of &?

Reply:
yeah I wasnt sure if this was the best to touch this file or copy over code and change it in another file

Reply:
I think we can just touch this file.